plugins/modules/system/sefcontext.py

```diff
@@ -22,6 +22,13 @@
     type: str
     required: yes
     aliases: [ path ]
+  equiv:
+    description:
+    - set target selinux equivalence to this path.
+    - setype must be set, the bogus 'many_t' works.
```

Comment:
```suggestion
    - I(setype) must be set, the bogus C(many_t) works.
```
See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation for how to format things.

Reply:
Feel free to make any doc changes you need.

Reply:
@dithotxgh please take a look at the conventions used in the code and documentation - the PR will not be merged otherwise.